src/Tolerance/Bridge/Symfony/Bundle/ToleranceBundle/DependencyInjection/Configuration.php

```diff
@@ -30,6 +30,7 @@ public function getConfigTreeBuilder()
             ->append($this->getOperationRunnersNode())
             ->append($this->getMetricsNode())
             ->booleanNode('operation_runner_listener')->defaultTrue()->end()
+            ->booleanNode('guzzle')->defaultFalse()->end()
```

Comment:
Could be removed and replaced by a new section in the documentation, but provide a simple "plug'n'play" feature.